test/stub_server.go

```diff
@@ -54,24 +57,31 @@ func NewStubServer(testSuite *RabbitTestSuite) *StubServer {
 	return stubServer
 }
 
-// Finds the relevant endpoint stub (if any), and calls its handler function
-func (stubServer *StubServer) handleRequest(req server.Request) (server.Response, error) {
-	stubServer.stubsMutex.RLock()
-	defer stubServer.stubsMutex.RUnlock()
+// StubResponse is a convenience method to quickly set up stubs that return a fixed value
+func (stubServer *StubServer) StubResponse(serviceName, endpoint string, returnValue proto.Message) {
+	stubServer.stubResponseAndError(serviceName, endpoint, returnValue, nil)
+}
 
-	// determine which endpoint to use
-	for _, stub := range stubServer.stubs {
-		if stub.ServiceName == req.ServiceName() && stub.Endpoint == req.Endpoint() {
-			return stub.Handler(req)
-		}
-	}
+// StubError is a convenience method to stub out a service error
+func (stubServer *StubServer) StubError(serviceName, endpoint string, err error) {
```

Comment:
Convention for `func` signatures here would probably be the below, but not important :smile: 

```go
func (s *StubServer) StubError(serviceName, endpoint string, err error) {
```